packages/universal-test-runner-types/package.json

```diff
@@ -11,8 +11,7 @@
     "prepack": "npm run clean && npm run compile"
   },
   "files": [
-    "dist/lib/src",
-    "dist/types/src",
+    "dist/lib",
```

Comment:
Should we just include all of `dist`? The file structure I'm seeing locally is:
```
dist/
  lib/
    index.js
  types/
    index.d.ts
```
so I think if we only include dist/lib, we won't get the index.d.ts file in the tarball.

Reply:
🤦 yes

Reply:
FIxed. Glad you're actually reading the PR 😆

Reply:
#229 FYI